contents/handbook/strategy/product-prioritization.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Product Prioritization
+sidebar: Handbook
+showTitle: true
+---
+
+PostHog aims to provide all the tools that our ICP needs in one platform. This is a _lot_ of products.
+
+## How do we prioritize?
+
+Products we build into the platform should:
+
+* be recognizable/positioned as products that our ICP already uses
+* use or add to customer or product data
+* improve our other products
+* be designed to enable us to get in first with that product (even if in practise we often rip and replace an existing product)
+* help customers build more successful products (this doesn't _just_ mean writing code, it means commercial stuff too)
+* ideally work with our snippet
```

Comment:
@timgl this last point was your suggestion from a conversation we had. Could you give me the reason _why_ you believe this? If not I'll remove it. I'm struggling to really justify it.

Reply:
Because it means people can turn on something like error monitoring by just going to posthog and toggling a switch, rather than having to implement another snippet.